prometheus/prometheus.go

```diff
@@ -1,6 +1,8 @@
 package prometheus
```

Comment:
Do we have a plan on how we are going to consume these metrics?

Reply:
Not yet. Still, they should still be available for scraping if needed by whomever deploys the operator.